src/concept.py

```diff
@@ -51,8 +58,12 @@ class Concept(BaseModel):
     related_concepts: list[WikibaseID] = Field(
         default_factory=list, description="List of related concept IDs"
     )
-    definition: Optional[str] = Field(
-        default=None, description="The definition of the concept"
+    ancestry: Optional[list[WikibaseID]] = Field(
```

Comment:
Is 'ancestry' a widely-used term? I needed to read this PR to figure out what it meant. If it's not, it could be replaced by something more descriptive like `recursive_parents`? (Not too happy with that name either, but i think it expresses the point i'm trying to make!)

Reply:
it's not afaik, but i've used the word a few times in internal discussions and it always seems help make the idea click! will have a think about alternatives though

Reply:
i've replaced `ancestry` with `recursive_parent_concepts` for the time being - just shout if you have a better idea before this is merged!

Reply:
lgtm!